packages/react-router/server/index.ts

```diff
@@ -62,9 +62,10 @@ export type {
   SerializeFrom,
   ServerBuild,
   ServerEntryModule,
-  ServerRuntimeMetaArgs,
-  ServerRuntimeMetaDescriptor,
-  ServerRuntimeMetaFunction,
+  // Aliases previously provided by runtime packages
+  ServerRuntimeMetaArgs as MetaArgs,
+  ServerRuntimeMetaDescriptor as MetaDescriptor,
+  ServerRuntimeMetaFunction as MetaFunction,
```

Comment:
Just calling out that this API is changing since it was being aliased in `@react-router/node` ☝️